File: orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/CloudbreakOrchestratorCancelledException.java
package com.sequenceiq.cloudbreak.orchestrator;

public class CloudbreakOrchestratorCancelledException extends CloudbreakOrchestratorException {

    public CloudbreakOrchestratorCancelledException(String message) {
        super(message);
    }

    public CloudbreakOrchestratorCancelledException(String message, Throwable cause) {
        super(message, cause);
    }

    protected CloudbreakOrchestratorCancelledException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }

    public CloudbreakOrchestratorCancelledException(Throwable cause) {
        super(cause);
    }
}

File: orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/exception/CloudbreakOrchestratorCancelledException.kt
package com.sequenceiq.cloudbreak.orchestrator.exception

class CloudbreakOrchestratorCancelledException : CloudbreakOrchestratorException {

    constructor(message: String) : super(message) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }

    protected constructor(message: String, cause: Throwable, enableSuppression: Boolean, writableStackTrace: Boolean) : super(message, cause, enableSuppression, writableStackTrace) {
    }

    constructor(cause: Throwable) : super(cause) {
    }
}
